P2PEnabledConference.js

```diff
@@ -0,0 +1,947 @@
+/* global __filename */
+
+import { getLogger } from 'jitsi-meet-logger';
+import * as JingleSessionState from './modules/xmpp/JingleSessionState';
+import JitsiConference from './JitsiConference';
+import * as JitsiConferenceEvents from './JitsiConferenceEvents';
+import * as RTCEvents from './service/RTC/RTCEvents';
+import * as XMPPEvents from './service/xmpp/XMPPEvents';
+
+const logger = getLogger(__filename);
+
+/**
+ * A peer to peer enabled conference that will try to use direct connection when
+ * available in case there are only 2 participants in the room. The JVB
+ * connection will be kept alive and it will be reused if the 3rd participant
```

Comment:
that is my understanding as well, but I thought that when it comes to order it should come with "the"